tooling/cli/src/build.rs

```diff
@@ -156,6 +156,17 @@ pub fn command(mut options: Options, verbosity: u8) -> Result<()> {
       if config_.tauri.bundle.appimage.bundle_media_framework {
         std::env::set_var("APPIMAGE_BUNDLE_GSTREAMER", "1");
       }
+
+      // TODO: Consider using the plugin crate directly
```

Comment:
not sure what this means but the way it is implemented right now is okay (until a better approach to extend the bunder)

Reply:
i meant importing the crate to be able to use the actual config struct. but yeah, not worth the headache here.